app/public/js/system/userConfig.js

```diff
@@ -49,4 +49,38 @@ userConfig.saveWindow = function(width, height) {
         window.localStorage.width = Math.round(width);
         window.localStorage.height = Math.round(height);
     }
-};
\ No newline at end of file
+};
+
+userConfig.scaleWindow = function(scale){
+    if("plus" === scale) {
```

Comment:
you should define the currentScale and newScale vars at the top and then re-assign.

So before the conditions start:
```javascript
 var currentScale;
 var newScale;
```
then you can just do ` currentScale = whateverValueYouWant`